azurerm/internal/tf/suppress/json.go

```diff
@@ -0,0 +1,27 @@
+package suppress
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func JsonDiff(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
one last comment, is there a reason not to use `structure.SuppressJsonDiff` here?

Reply:
only my lack of knowledge, that such method exists ;-) I'll remove `suppress/json*` stuff and fall back to `structure.SuppressJsonDiff`

Reply:
Done.